aristanetworks/ansible-avd
# Create "Table Of Contents" automatically for Fabric and Device Documentation

## Change Summary
Create "Table Of Contents" automatically for Fabric and Device Documentation
## Types of changes


## Related Issue(s)



## Component(s) name

`arista.avd.eos_designs`
`arista.avd.eos_cli_config_gen`

## Proposed changes

* Adding new module "add_toc" which is an ansible wrapper around md_toc python library
* Changing Device and Fabric Documentation to use "add_toc" module
* Changed Header levels in Fabric Documentation to be similar to Device Documentation
* Update requirements.txt

The purpose of this change is to allow us to simplify the Device Documentation later.

## How to test


Run test repo and it produces almost the same TOC as we had done manually. The changes are bugs we hadn't noticed, so this change fixes those as well.

Run Molecule and see only TOC changes.

## Checklist:
